src/libraries/Common/tests/System/Net/Http/PostScenarioTest.cs

```diff
@@ -262,7 +249,15 @@ public async Task PostAsync_EmptyContent_ContentTypeHeaderNotSent(Configuration.
 
                     // Compute MD5 of request body data. This will be verified by the server when it
                     // receives the request.
-                    requestContent.Headers.ContentMD5 = TestHelper.ComputeMD5Hash(requestBody);
+                    if(PlatformDetection.IsBrowser)
+                    {
+                        // [ActiveIssue("https://github.com/dotnet/runtime/issues/37669", TestPlatforms.Browser)]
+                        requestContent.Headers.Add("Content-MD5-Skip", "browser");
```

Comment:
Why modify the tests for this issue, instead of skipping for now?

Reply:
It was blocking most POST tests. They are more useful in this reduced form, than not running it at all.

Reply:
IMHO, these test changes should be in a separate PR. This one is enabling tests that depend on echo servers, IIUC.